### PR TITLE
perf(ui): ui twick & update theme-related config

### PR DESCRIPTION
**Pull request by CharlesChiuGit**
Add some nice visual effects based on current plugins.
1. `wilder`
wilder

2. `indent-blacnkline`
indent-line

3. `virtual-text`
v-text

### CODE REVIEW

**Comment by ayamir** [comment]
I have just be diagnosed with COVID-19 and need to rest for a few days.

File: lua/modules/tools/config.lua [hunk 1]
```diff
@@ -225,8 +225,14 @@ function config.wilder()
 		),
 	})
 
+	local match_hl = require("utils").hlToRgb("String", false)
```
Comment (Jint-lzxy) [inline lua/modules/tools/config.lua:228]: In this case, we no longer need the `utils` module. This line can be replaced as follows:
```suggestion
	local string_fg = vim.api.nvim_get_hl_by_name("String", true).foreground
	local match_hl = string_fg ~= nil and string.format("#%06x", string_fg) or "#ABE9B3"
```
Note that `String` is not one of the builtin hl groups. We need to set a fallback value for it.

**Review by Jint-lzxy — CHANGES_REQUESTED** [review]
IMO I will accept the changes for `wilder`. However, for `indent-blankline` and `virtualtext`s, they are **intentionally** _not_ background highlighted as this would affect performance when too many such objects are rendered in the same screenview _(for example, scrolling quickly will cause deferred rendering)_.

https://github.com/ayamir/nvimdots/blob/708c1db3595bb3ff2878d24cea243923c4fd423c/lua/modules/ui/config.lua#L251-L255

**Comment by CharlesChiuGit** [comment]
@Jint-lzxy fair enough, will fix.

**Comment by CharlesChiuGit** [comment]
@ayamir take care!

File: lua/modules/tools/config.lua (hunk 1 above)
Comment (CharlesChiuGit) [reply lua/modules/tools/config.lua:228]: Ok, will do.

**Comment by CharlesChiuGit** [comment]
@Jint-lzxy PTAL.

**Review by Jint-lzxy — APPROVED** [review]
LGTM 🎉 Thank you for your efforts!

**Pull request merged**